deploy/helm/sumologic/values.yaml

```diff
@@ -1537,3 +1537,20 @@ otelcol:
           receivers: [jaeger, opencensus, otlp, zipkin]
           processors: [memory_limiter, k8s_tagger, source, resource, batch, queued_retry]
           exporters: [zipkin]
+
+## Configure telegraf-operator
+## This is an experimental configuration and shouldn't be used in production environment
+## ref: https://github.com/influxdata/helm-charts/blob/master/charts/telegraf-operator/values.yaml
+telegraf-operator:
+  enabled: false
+  replicaCount: 1
```

Comment:
@sumo-drosiek what is the replicatCount about?

Reply:
about replicas 🤷 . It's 3 by default. AFAIK telegraf-operator performs leader election